packages/default-storage/example/examples/Functional.tsx

```diff
@@ -130,9 +129,15 @@ function Functional(): JSX.Element {
         }
       })
       .then(async () => {
-        const AsyncStorageTestSupport = TurboModuleRegistry
-          ? TurboModuleRegistry.get("AsyncStorageTestSupport")
-          : NativeModules["AsyncStorageTestSupport"];
```

Comment:
@tido64 when using your example of the `AsyncStorageTestSupport` type, I was getting a weak typing issue saying `AsyncStorageTestSupport` had no properties in common with `TurboModule` - which is why I added the `getConstants` field. 

Also, I set the AsyncStorageDelegate to have a boolean parameter here so that the `AsyncStorageTestSupport.test_setDelegate(resolve);` calls in the Promises below would work. Is there a better way to do this?

Reply:
You probably want this: `type AsyncStorageTestSupport = TurboModule & {`